### ISSUE TITLE
mu symbol not detected for alt text when using exponent: "eng"

### ISSUE THREAD

**Issue by Ants-Aare** [Mc-Zen/zero#97]
```
#zero.set-num(exponent:"eng")
#zero.zi.m(50e-6)
```

this should display but it's giving an error with missing alt text. assertion failed: Failed to auto-generate alt description for unit component "μm". Please provide a manual alt text for this unit.

this works just fine for millimetres and nanometres though, and unintuitively for manually declared micrometers as well

```
#let um = zero.zi.declare("µm")

#zero.set-num(exponent:"eng")
#zero.zi.m(50e-3)
#um(50)
#zero.zi.m(50e-9)
```

it seems to fail at this line, with prefix in prefixes returning false for the version with the mu added by the eng exponent but returning true for the version with the mu added by declaring it as a unit:
```
if (prefix in prefixes) and (unit in units) {
  return join-prefix-unit(prefixes.at(prefix), get-unit(unit), text.lang)
}
```

the fix I found was to replace the mu line in the prefixes with this:
```
#let prefixes = (
  en: (
    a: "atto",
    f: "femto",
    p: "pico",
    n: "nano",
    "µ": "micro",
    "μ": "micro",
    ...
```
apparently there are two different mu symbols in circulation(?)

**Comment by Mc-Zen** [comment]
Thank you @Ants-Aare for catching this before the release 🙏

Yes there are actually two distinct Unicode symbols for this, the micro symbol (U+00B5) and the Greek mu (U+03BC). I must have mixed them up again somewhere.

**Comment by Mc-Zen** [comment]
I wonder which is even the correct one. 

`$upright(mu)"m"$ $"µm"$`

<img width="309" height="101" alt="Image" src="https://github.com/user-attachments/assets/f8c119cd-290b-4006-addf-c646a6375882" />

The first one is the greek mu, the second one the micro symbol. Technically, from a unicode-perspective the micro symbol is correct because it was designed just for that purpose. 

Edit, yes I think the second one is the correct one. In line 365 of units.typ, I put `sym.mu` which is the wrong (greek) one.

**Comment by Ants-Aare** [comment]
I would use the micro symbol for everything then. it looks a lot better in your example image. I'm guessing the greek one is for math related contexts or the [Muon particle](https://en.wikipedia.org/wiki/Muon).

**Comment by hpcfzl** [comment]
> I wonder which is even the correct one.

https://github.com/Mc-Zen/zero/blob/main/tests/accessibility/test.typ#L210

I added that comment due to https://github.com/Mc-Zen/zero/pull/90#issuecomment-4925220340.

**Comment by Mc-Zen** [comment]
Ahaaa, was that why the test failed for you at first?

**Comment by hpcfzl** [comment]
Well in a way, yes, the `unit-component-description()` call would error.
